homopolymer_fit/Ising_fitter_homopolymer.ipynb

```diff
@@ -61,112 +63,139 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 18,
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def extract_cd_signal_from_aviv(filepath):\n",
```

Comment:
putting these blocks into functions will help other people follow the code more easily. I'll continue doing this in the next set of changes. This will also improve the "componentization" and "modularity", meaning we can import them instead of copy/paste-ing the same code